WISE-Community/WISE
# Upgraded NotebookService from AngularJS to Angular

Upgraded NotebookService to Angular and wrote tests for NotebookService.

To test:
- Make sure Notebook tools all work as before:
  - Student notebook and report in vle (preview + runs)
  - Teacher notebook and notebook grading view in classroom monitor
  - Notebook authoring

Closes #2479.
